test/backend/test_dynamic_shape.py

```diff
@@ -248,6 +248,32 @@ def test_conv_transpose(self):
 
     np.testing.assert_almost_equal(tf_model_output[0], test_output, decimal=5)
 
+  def test_depth_to_space(self):
+    b,c,h,w = shape = [2, 48, 5, 6]
```

Comment:
The code looks fine. Please run yapf as described in https://github.com/onnx/onnx-tensorflow#code-standard to keep the code format and style consistent. Thanks.

Reply:
Thank you for pointing out that document, done on both files.